src/components/button/button.twig

```diff
@@ -1,10 +1,27 @@
 {% set tag_name = tag_name|default(href ? 'a' : 'button') %}
+{# Check for 'content_start' and 'content_end' #}
+{% set _content_start = block('content_start') %}
+{% set _content_end = block('content_end') %}
+
+{# Set a default icon if aria-pressed is set and no icon (or block content) was supplied #}
+{% if
+  aria_pressed and
+  content_start_icon is empty and
+  content_end_icon is empty and
+  _content_start|trim is empty and
+  _content_end|trim is empty
+%}
+  {% set content_start_icon = 'notifications' %}
+{% endif %}
```

Comment:
If the `aria_pressed` value stays as a `string` value, I might want to check specifically for `'true'` and/or `'false'`. Otherwise, a value of `asdf` (or any random string) will add the notifications icon, which doesn't seem correct.

Reply:
:+1: Yeah I think we'll want to check the value of `aria_pressed`

Reply:
Udpated! https://github.com/cloudfour/cloudfour.com-patterns/pull/1298/commits/22724ac138627e739cc651043201d6d8b02f2e48